app/views/users/show.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="container user-profile">
+  <div row="row">
+    <div class="col-md-offset-2 col-md-8 col-lg-offset-3 col-lg-6">
+      <div class="well profile">
+        <div class="col-sm-12">
+          <div class="col-xs-12 col-sm-8">
+            <h2><%= current_user.name %></h2>
+            <p><strong>Email:</strong> <%= current_user.email %></p>
+          </div><!-- end of col-xs-12 -->
+          <div class="col-xs-12 col-sm-4 text center">
+            <figure>
+              <img src="http://www.localcrimenews.com/wp-content/uploads/2013/07/default-user-icon-profile.png" alt="" class="img-circle img-responsive">
```

Comment:
All assets should be served from the app. 

If you want to use this image put a copy of it in app/assets/images and use `image_tag` helper.

Currently with this `http`, non-SSL URL, browsers will complain with mixed content security warnings when the production app is viewed on an `https://` URL.